src/domTree.js

```diff
@@ -65,26 +176,11 @@ export class HtmlDomContainer<ChildType: VirtualNodeInterface>
         options?: Options,
         style?: CssStyle,
     ) {
-        this.classes = classes || [];
-        this.children = children || [];
-        this.attributes = {};
-        this.height = 0;
-        this.depth = 0;
-        this.maxFontSize = 0;
-        this.style = Object.assign({}, style);
-        if (options) {
-            if (options.style.isTight()) {
-                this.classes.push("mtight");
-            }
-            const color = options.getColor();
-            if (color) {
-                this.style.color = color;
-            }
-        }
+        initNode.call(this, classes, children, options, style);
```

Comment:
Does this set the prototype correctly?

Reply:
@kevinbarabash `initNode` doesn't touch prototypes but just initializes properties.